Source/Core/Core/AchievementManager.h

```diff
@@ -206,4 +205,31 @@ class AchievementManager
   std::recursive_mutex m_filereader_lock;
 };  // class AchievementManager
 
+#else  // USE_RETRO_ACHIEVEMENTS
+
+#include <string>
+
+namespace DiscIO
+{
+class Volume;
+}
+
+class AchievementManager
+{
+public:
+  static AchievementManager& GetInstance()
```

Comment:
Can this one be `constexpr` as well, and basically remove every mention of it during compilation? Or is that too much to ask of `constexpr`?

Reply:
I tried it now to make sure, and it won't compile.

Reply:
Ah well, was worth a try. Looks good otherwise 👍